docs/articles/documentation/why-testcafe/README.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: docs
+title: Why TestCafe?
+permalink: /documentation/why-testcafe/
+---
+
+# Why TestCafe?
+
+TestCafe excels where other end-to-end testing solutions fall short.
+
+## No 3rd-Party Dependencies
+
+TestCafe does not depend on Selenium or other testing software. TestCafe runs on the popular Node.js platform, and makes use of the browsers that you already have. For many users, the installation process consists of a single step:
+
+```js
+npm install -g testcafe
+```
+
+## Human-readable JavaScript tests
+
+The core components of our Node.js API are easy to use, even with little to no prior knowledge of JavaScript.
+
+## A first-class citizen in the Node.js ecosystem
+
+Like all Node.js scripts, TestCafe tests can leverage the capabilities of third-party JavaScript libraries and preprocessors.
+
+## No compromises
+
+The advanced features of TestCafe help you test complex, security-sensitive web applications.
+
+### Mock Requests
+
+Emulate HTTP responses to feed sample data to your app, troubleshoot connectivity errors, and cheat downtime.
+
+### Client-side injections
+
+Execute custom client-side code to parse the web page, examine its state, or even add extra dependencies.
+
+### Frames and windows
+
+Make use of multiple browser windows to test out complex user interactions. Switch to and from iframes with ease and stability.
+
+### Concurrent tests
+
+Run your tests in multiple browsers at once to quickly uncover browser-specific errors.
+
+## Made with automation in mind
+
+### Automated waiting
+
+Manual timeouts are a thing of the past. Asynchronous from the ground up, TestCafe knows when to wait and what to wait for before a test can continue.
+
+### Automated authentication scripts
+
+Save and activate user roles to easily switch between different user accounts.
+
+### Hooks
+
+Keep your code clean with hooks — routines that are repeated before and after each test.
+
+## Helps you help yourself
+
+### TestCafe Studio
+
+The GUI and the test recorder built into TestCafe Studio make our product accessible to people with zero programming experience.
+
+### TestCafe Live
+
+Enable the built-in [Live Mode](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/run-tests.html#live-mode) to reload your test every time you make changes to the test file.
```

Comment:
I don't think 'reload' is the right word. Makes me think that I change the test file, then that file is reloaded... Should we probably say "re-run", or "restart". 

```suggestion
Enable the built-in [Live Mode](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/run-tests.html#live-mode) to restart your test every time you make changes to the test file.
```